example/src/screens/MainScreen.tsx

```diff
@@ -74,11 +99,76 @@ export default class MainScreen extends Component<any, MainScreenState> {
       .then((response) => response.data);
   }
 
+  componentDidMount() {
+    this.updateTrackingConsent()
+  }
+
+  updateTrackingConsent() {
+    getTrackingConsent().then(consent => {
+      this.setState({
+        trackingConsent: consent
+      })
+    })
+  }
+
+  setTrackingConsentModalVisible(visible: boolean) {
+    this.setState({ trackingConsentModalVisible: visible })
+  }
+
+  renderTrackingConsentItem({ item }) {
+    const isSelected = TrackingConsent[item] === this.state.trackingConsent
```

Comment:
Same as for modal there are examples in the `shopist-reactnative` application in how to render a list item from an external component: https://github.com/DataDog/shopist-reactnative/blob/master/app/components/cart/CartItem.js

Reply:
done